### DISCUSSION TITLE
Where to get the Mac version

### DISCUSSION THREAD

**Discussion by DaemonLion** [clementine-player/Clementine#7361] [Q&A]
Hi Everyone!

It seems that I no longer able to find a place to download the Mac version of Clementine, is it no longer available?

**Comment by colindavey** [comment]
I have the same question!

**Reply by timelawd** [reply]
gfgfh

**Comment by awais4212** [comment]
The official Clementine website still lists a Mac OS X version of the player. 

In the GitHub releases of the project, you can see the source code and some asset builds. 

There has been a recent non-release candidate version (1.4.1) after many years without updates. 

Clementine is flagged as end-of-life / discontinued in some Mac app directories (e.g. RoaringApps shows it’s no longer maintained)

**Reply by timelawd** [reply]
Can you please tell me where to find this "recent non-release candidate version (1.4.1)"?
Thank you very much for your reply.
Best regards.
